src/algebra/lie/abelian.lean

```diff
@@ -186,14 +186,19 @@ def maximal_trivial_linear_map_equiv_lie_module_hom :
 { to_fun    := λ f,
     { map_lie' := λ x m, by
       { have hf : ⁅x, f.val⁆ m = 0, { rw [f.property x, linear_map.zero_apply], },
-        rw [bracket_apply, sub_eq_zero, ← linear_map.to_fun_eq_coe] at hf, exact hf.symm, },
+        rw [lie_hom.lie_apply, sub_eq_zero, ← linear_map.to_fun_eq_coe] at hf, exact hf.symm, },
       ..f.val, },
   map_add'  := λ f g, by { ext, simp, },
   map_smul' := λ F G, by { ext, simp, },
   inv_fun   := λ F, ⟨F, λ x, by { ext, simp, }⟩,
   left_inv  := λ f, by simp,
   right_inv := λ F, by simp, }
 
+@[simp] lemma coe_maximal_trivial_linear_map_equiv_lie_module_hom
```

Comment:
Hmm, the name of this lemma seems fine, but the name of the lemma below is a bit confusing, since the `_apply` suggests that it is about functions apply to some element. I think I would replace the `_apply` by a `'` there.

Reply:
Good point. I've renamed the two lemmas below, dropping `_apply` suffix and changing the prefix from `coe_` to `coe_linear_map_`.